appveyor.yml

```diff
@@ -57,6 +57,7 @@ environment:
   # "(converted from warning) unable to identify current timezone 'C':" for an unknown reason.
   # This environment variable works around to test SparkR against a higher version.
   R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+  MAVEN_OPTS: -Xss256m
```

Comment:
256MB as a thread stack size sounds very large - the default is 1MB. Does it need to be that high?

Reply:
Maybe not .. let me debug a bit more.